### PR TITLE
Correct Wazuh docs on secrets encryption

### PR DESCRIPTION
**Pull request by MoteHue**
Also fixes some double quotes that were the wrong type of quote

### CODE REVIEW

**Comment by jackhodgkiss** [comment]
Deployments of newer Wazuh such as the one defined in `requirements.yml` does not use elasticsearch.

`/var/ossec` and possibly `/var/lib/wazuh-indexer` are the paths you create separate mounts for.

File: doc/source/configuration/wazuh.rst [hunk 1]
```diff
@@ -73,7 +73,7 @@ Optional: define LVM volumes ``etc/kayobe/inventory/group_vars/wazuh-manager/lvm
           size: "100%VG"
           filesystem: "ext4"
           mount: true
-          mntp: “/var/lib/elasticsearch”
+          mntp: "/var/lib/elasticsearch"
```
Comment (markgoddard) [inline doc/source/configuration/wazuh.rst]: @jackhodgkiss actually ended up mounting `/var/ossec`, claiming it has greater storage requirements. Perhaps we actually need both?

**Comment by MoteHue** [comment]
Cheers @jackhodgkiss, @markgoddard, I've proposed a change to suggest these different LVM configs instead

File: doc/source/configuration/wazuh.rst [hunk 2]
```diff
@@ -73,7 +75,7 @@ Optional: define LVM volumes ``etc/kayobe/inventory/group_vars/wazuh-manager/lvm
           size: "100%VG"
           filesystem: "ext4"
           mount: true
-          mntp: “/var/lib/elasticsearch”
+          mntp: "/var/ossec"
```
Comment (markgoddard) [inline doc/source/configuration/wazuh.rst:78]: Let's go with this for now and keep an eye on /var/lib/wazuh-indexer.

**Review by markgoddard — DISMISSED** [review]

**Comment by markgoddard** [comment]
pep8 says trailing whitespace in docs

**Review by markgoddard — APPROVED** [review]

**Pull request merged**